app/src/test/java/org/mozilla/focus/utils/UrlUtilsTest.kt

```diff
@@ -186,4 +186,27 @@ class UrlUtilsTest {
         assertEquals("random.mozilla.org", UrlUtils.stripCommonSubdomains("random.mozilla.org"))
         assertEquals(null, UrlUtils.stripCommonSubdomains(null))
     }
+
+    @Test
+    fun testToDisplayUrl() {
+        infix fun String.shouldBecome(output: String) {
```

Comment:
This is modifying our assertion style and it's potentially easy to overlook: I would go back to calling `assert*` directly or encapsulating that style in a helper function, e.g. `assertToDisplayUrlIsUnchanged(input)`.

Two other options to keep syntax clean are:
- Put the assertions in a loop. This complicates test code and obfuscates line numbers though
- Create a parameterized test. I haven't tried this before and JUnit's syntax seems verbose, but it solves the complexity of loops. This also encourages the style of unit testing where you have one unit test for each "case".

Parameterized tests would look something like:
```kotlin
@Test
@Parameters("error:-8", "hello world")
fun `test displayUrl is unchanged for X style parameter`(input: String) {
    assertEquals(input, UrlUtils.toDisplayUrl(input)
}
```

If you're interested in changing our assertion style, I'd propose it to the team. Personally, I've been missing jasmine's (it's behavior driven, I think?) hierarchical test setup and naming structure: it works well with GIVEN, WHEN, THEN.